main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/rs/zerolog"
+
+	"github.com/ovalfi/go-sdk/api"
+	"github.com/ovalfi/go-sdk/model"
+	"github.com/ovalfi/go-sdk/model/example"
+)
+
+func main() {
+	_ = os.Setenv("TZ", "Africa/Lagos")
```

Comment:
we might not need this or rename it as `example.go` - this is a public API

Reply:
I don't get this please

Reply:
remove the line.